src/options/quantifiers_options.toml

```diff
@@ -1316,6 +1316,30 @@ header = "options/quantifiers_options.h"
   default    = "false"
   help       = "use satisfiability check to verify correctness of candidate rewrites"
 
+[[option]]
+  name       = "sygusRewSynthInput"
+  category   = "regular"
+  long       = "sygus-rr-synth-input"
+  type       = "bool"
+  default    = "false"
+  help       = "synthesize rewrite rules based on the input formula"
+
+[[option]]
+  name       = "sygusRewSynthInputNVars"
+  category   = "regular"
+  long       = "sygus-rr-synth-input-nvars=N"
+  type       = "int"
+  default    = "3"
```

Comment:
why is this the default?

Reply:
this is a tradeoff between efficiency and generality, 2 is not general enough, 4 is too inefficient.

Reply:
This should be said somewhere, probably in the header of the class using the option